spec/live/cbp_support.rb

```diff
@@ -0,0 +1,37 @@
+require 'core/spec_helper'
+require 'core/cbp_helper'
+describe 'Endpoints that support CBP' do
+  describe ZendeskAPI::Group do
+    describe '/groups' do
+      expect_cbp_response_for(client.groups)
+      expect_cbp_response_for(client.groups.assignable)
+    end
+  end
+
+  describe ZendeskAPI::GroupMembership do
+    describe '/groups/{id}/memberships' do
```

Comment:
sigh, there's some code repetition here... since the test makes two requests instead of one.. .not sure how to get around this... still thinking about it